app/app.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: nodejs16
```

Comment:
Node active support LTS is version 20 now.

Does the React app work with Node version 20? (If not, I can look into getting that repaired.)